src/components/progress-flow/examples/progress-flow-alternative-ui.tsx

```diff
@@ -0,0 +1,99 @@
+import { Component, h, State } from '@stencil/core';
+import { FlowItem } from '../progress-flow.types';
+
+/**
+ * Alternative UI
```

Comment:
I'm bringing this into a thread 😄

> This is how it looks today and it is also explained when it should be used. What do you think about it now @adrianschmidt? 🙂
> ![Screenshot 2021-08-17 at 10 55 04](https://user-images.githubusercontent.com/19202946/129695738-d69a6301-3181-49e4-9e46-82cb8fc5e1aa.png)

Ah! Ok, yeah, if it is functionally different, then I think including it is reasonable.

I don't understand why the helper class is called `has-detached-steps` though? "Detached steps" sounds like the same thing as "off-progress steps" to me.

Also, I think we should consider making this a property instead of a helper-class, and force the component to be read-only when that property is used. Otherwise this isn't an "alternative UI", but an "alternative look", and people will use it as that…

@Kiarokh

Reply:
> I'm bringing this into a thread 😄
Awesome! 😄 

> helper class is called `has-detached-steps` though?

You're right. It's perhaps not the best name. However, something like "alternative-design" or "alternative-ui" isn't neither. In case we add a new alternative ui for this component in future, then we should call that "alternative-ui-2" or something 😅

Open for suggestions. I prefer some name that describes the way it looks visually. 


> we should consider making this a property instead of a helper-class,

I think it's a good idea to have properties actually, rather than helper classes. We can be more consistent about this for all components. Today, we sometimes use helper classes for things that change the visual look and feel, and sometimes use properties which I think is very inconsistent.

So I think it's up to us to make a convention and follow it in all components for simplicity. If you want we can start here and make both `is-narrow` and `has-detached-steps` (or whatever we call it) properties.

I think it may be more natural for consumers to use properties. What do you suggest?